manifests/s/stevencohn/OneMore/5.7.2/stevencohn.OneMore.locale.en-US.yaml

```
@@ -0,0 +1,35 @@
+# Created with YamlCreate.ps1 v2.2.0 using InputObject 🤖 $debug=QUSU.CRLF..7-2-9
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: stevencohn.OneMore
+PackageVersion: 5.7.2
+PackageLocale: en-US
+Publisher: River
+PublisherUrl: https://stevencohn.net/
+PublisherSupportUrl: https://github.com/stevencohn/OneMore/issues
+# PrivacyUrl:
+Author: Steven M. Cohn
+PackageName: OneMoreAddIn
+PackageUrl: https://github.com/stevencohn/OneMore
+License: MIT License
```

```suggestion
License: MPL-2.0
```
Product is licensed under [MPL-2.0](https://github.com/stevencohn/OneMore/blob/main/LICENSE)